### ISSUE TITLE
Fuser dies with "Unknown FUSE opcode (40)"

### ISSUE THREAD

**Issue by mdbooth** [cberner/fuser#143]
This is apparently FUSE_POLL, and I note that https://github.com/cberner/fuser/issues/16 is already open to add support for that.

I can trigger this trivially with less's tail mode:
```
$ less myfs/foo.log
<press F>
!boom!
```

As we can reasonably expect more opcodes to be added in the future and fuser may not support all of them immediately, please could it warn but not crash?

**Comment by cberner** [comment]
I'm not able to reproduce this with the `simple.rs` example and less version 551. What fuser version are you using?

**Comment by mdbooth** [comment]
I have "0.7.0" in Cargo.toml.

I wonder if it's a kernel version thing? I'm running Fedora 34 on WSL2: 
```
$ uname -r
4.19.104-microsoft-standard
```

**Comment by cberner** [comment]
I looked through the code path for this error, and I think this is already fixed on master. Can you try master and see if it's fixed for you? I still haven't been able to reproduce the crash, but I can cut a new release if the issue is fixed

**Comment by cberner** [comment]
I believe this is fixed in 0.8.0. Let me know if not though
